ui/hooks/accounts/useMultichainClient.test.ts

```diff
@@ -0,0 +1,73 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { HandlerType } from '@metamask/snaps-utils';
+import { BtcAccountType, BtcMethod } from '@metamask/keyring-api';
+import { MultichainNetworks } from '../../../shared/constants/multichain/networks';
+import { BITCOIN_WALLET_SNAP_ID } from '../../../shared/lib/accounts/bitcoin-wallet-snap';
+import { SOLANA_WALLET_SNAP_ID } from '../../../shared/lib/accounts/solana-wallet-snap';
+import {
+  handleSnapRequest,
+  multichainUpdateBalance,
+} from '../../store/actions';
+import { useMultichainClient, WalletClientType } from './useMultichainClient';
+
+jest.mock('../../store/actions', () => ({
+  handleSnapRequest: jest.fn(),
+  multichainUpdateBalance: jest.fn(),
+}));
+
+const mockHandleSnapRequest = handleSnapRequest as jest.Mock;
+const mockMultichainUpdateBalance = multichainUpdateBalance as jest.Mock;
+
+describe('useMultichainClient', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const mockAccount = {
+    address: 'tb1q2hjrlnf8kmtt5dj6e49gqzy6jnpe0sj7ty50cl',
+    id: '11a33c6b-0d46-43f4-a401-01587d575fd0',
+    options: {},
+    methods: [BtcMethod.SendMany],
+    type: BtcAccountType.P2wpkh,
+  };
```

Comment:
Being picky here, but we could move this "account" to each `testCases` since this one belongs to Bitcoin, but Solana would use a different kind of account, WDYT?

Reply:
Well spotted, makes sense.